utils/common/src/main/java/org/apache/brooklyn/util/text/Strings.java

```diff
@@ -916,4 +916,17 @@ public static String toUniqueString(Object x, int optionalMax) {
         return maxlenWithEllipsis(xs, optionalMax-8)+"/"+Integer.toHexString(xs.hashCode());
     }
 
+
+    /**
+     * Parses a string as comma separated values and returns a list of the entries with whitespace removed.
+     * @param csv
+     * @param separators
+     */
+    public static List<String> parseCsv(String csv, String separators) {
+        List<String> result = MutableList.of();
+        for (String value: csv.split(separators)) {
+            result.add(value.trim());
```

Comment:
Suggest adding some tests for this method. `.split` will return non-empty array for the empty string I believe